views/support/faqbrowse.tt

```diff
@@ -0,0 +1,109 @@
+<script language='JavaScript'>
```

Comment:
😬 This is extremely loathsome, but I don't think it'll _break,_ per se.

Reply:
Honestly I'm not sure we actually use the expand-in-place stuff, so I could probably just rip it out 🤔